### ISSUE TITLE
flow finetune grad_norm nan

### ISSUE THREAD

**Issue by AntroFyo** [FunAudioLLM/CosyVoice#1785]
Hi when i finetune flow model,  
the grad_nor is nan. 
The lr is set to 1e-5 and 1e-6.
<img width="409" height="349" alt="Image" src="https://github.com/user-attachments/assets/0f92d901-b27e-4f48-a810-ebb33083b836" />

**Comment by aluminumbox** [comment]
are you training on libritts? try remove --use-amp, the loss is much smaller than base model training, should be around 0.55-0.6

**Comment by AntroFyo** [comment]
Not libri. I am using an open-source data about 7k hours. And using -use-amp or not is of useless for grad_nore being NAN.
